Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.h

```diff
@@ -71,6 +71,7 @@ class RemoteGraphicsContextGLProxy
     void didReceiveInvalidMessage(IPC::Connection&, IPC::MessageName) final { }
 
     // WebCore::GraphicsContextGL overrides.
+    GCEGLSync createEGLSync(ExternalEGLSyncEvent) final;
```

Comment:
The main purpose of ExternalEGLSyncEvent is that we don't need platform-specific ifdefs for the arguments.
The main purpose we want that is that we then don't need the explicit manual function, rather we can just list this function in the autogen section and get the autogenerated implementation.